src/robusta/core/sinks/sink_base.py

```diff
@@ -7,6 +7,7 @@ def __init__(self, sink_params: SinkBaseParams):
         self.sink_name = sink_params.name
         self.params = sink_params
         self.default = sink_params.default
+        self.telemetry = None
```

Comment:
I like the idea to save it on the sink.
But I think telemetry is never initialized.
I would save a reference of `registry` rather than telemetry on the sink.
I think it's more generic, and might be used for other stuff by the sinks later on
I would pass the registry and save it on the sink in the constructor.
And I would add the telemetry to the registry